src/client/user/components/UserLinkTable/StatusBar/index.tsx

```diff
@@ -92,18 +88,45 @@ const StatusBar = () => {
 
   const dispatchCloseStatusBar = () => {
     dispatch(userActions.closeStatusBar())
+    // store last acknowledged message
+    localStorage.setItem('statusBarMessage', JSON.stringify(statusBarMessage))
   }
 
+  const getLatestJob = () => dispatch(userActions.getLatestJob())
+
+  useEffect(() => {
+    getLatestJob()
+  }, [])
+
+  useEffect(() => {
+    if (header && body) {
+      const localStatusBarMessage = localStorage.getItem('statusBarMessage')
+      const statusBarMessageString = JSON.stringify(statusBarMessage)
+      if (localStatusBarMessage === statusBarMessageString) {
+        // if current message is same as last acknowledged message, hide
+        setShowStatusBar(false)
+      } else {
+        // current message has not been acknowledged before
+        setShowStatusBar(true)
```

Comment:
i'm wondering what would happen if we decide to add in S3 bucket lifecycle rules? say a user does the following:
- User bulk uploads links, everything is successful and zipped QR codes are uploaded to S3
- The expiry time passes (e.g. 1 month), zip files are deleted from S3
- User clears local storage and logs into the website, sees the success modal to download zip files
- User tries to download the now non-existent zip files, gets error

is there any way to prevent showing the latest job after a certain amount of time has passed (e.g. a week)?

Reply:
Sure, Hena and I discussed that we could do something like this in the future (i.e. cut off the display after a certain time period e.g. one week) but we're not implementing this for now to give users full control of their experience. We can implement the lifecycle rules after implementing the time period cutoff. 

The technical implementation should be quite straightforward because we can just modify the call to retrieve the latest job for a user to cut off after a certain period.

Reply:
i see, thanks for clarifying!